src/main/java/AddCommand.java

```diff
@@ -1,31 +1,42 @@
+import java.util.InputMismatchException;
 
 public class AddCommand implements Command {
-    private final Book usersBook;
+    private final UserBook usersBook;
     private final Console console;
 
-    public AddCommand(Book usersBook, Console console) {
+    public AddCommand(UserBook usersBook, Console console) {
         this.usersBook = usersBook;
         this.console = console;
     }
 
     @Override
     public void execute() {
 
-        console.printLn("Enter your name: ");
-        String name = console.nextLine();
+        try {
+            console.printLn("Enter your name: ");
+            String name = console.nextLine();
 
-        console.printLn("Enter last name:");
-        String surName = console.nextLine();
+            console.printLn("Enter last name:");
+            String surName = console.nextLine();
 
-        console.printLn("Enter age:");
-        int age = console.nextInt();
+            console.printLn("Enter age:");
+            int age = console.nextInt();
 
-        User user = new User(name, surName, age);
-        usersBook.addUser(user);
+            console.printLn("Enter id");
```

Comment:
How client of your application will know id to enter ? He should know id of all users which was ever inserted in user book to create unique id every time.  When you register to website, have you seen if website ask to  enter id to register you ?